point_of_sale/overview/getting_started.rst

```diff
@@ -0,0 +1,154 @@
+
+Configure your payment methods
+------------------------------
+
+To add a payment method, you first need to create it. Go to
+:menuselection:`Point of Sale --> Configuration --> Payment Methods`
+and click on create. Then, you can name your payment method and set it
+up.
+
+.. image:: media/getting_started_02.png
+    :align: center
+    :alt: Creating a new payment method for a Point of Sale.
+
+.. note::
+   Don’t forget your credentials for methods using a payment terminal.
+
+Now, you can select the payment method in your PoS settings. To do so,
+go to :menuselection:`Point of Sale --> Configuration --> Point of
+Sale` and open the PoS you need the payment method for. Then, add the
+payment method.
+
+.. image:: media/getting_started_03.png
+    :align: center
+    :alt: Making the payment method available in a point of sale.
+
+.. note::
+   It’s also the menu where you can edit all your point of sale settings.
+   Some more features are available for restaurants.
+
+Your first PoS session
+======================
+
+Your first order
+----------------
+
+Now, you can open your first session. From the PoS dashboard, you can
+start a new session.
+
+.. image:: media/getting_started_04.png
+    :align: center
+    :alt: Launching a new session.
+
+After the loading screen, you arrive on the PoS interface.
+
+.. image:: media/getting_started_05.png
+    :align: center
+    :alt: Point of sale dashboard.
+
+Once an order is completed, you can register the payment. All the
+available ones appear on the left of the screen. Select the payment
+method and enter the received amount. Then, you can validate the
+payment.
+
+Return and refund products
+--------------------------
+
+Having a well-thought-out return policy is key to keep customers. Making
+it easy for you to accept and refund those returns is, therefore, also a
+key aspect of your PoS interface.
+
+From the PoS interface, select the product your customer wants to
+return, use the +/- button and enter the quantity they need to return.
+If they need to return multiple products, repeat the process.
+
+.. image:: media/getting_started_06.png
+    :align: center
+    :alt: Refunding a product with the point of sale app.
+
+When on the payment interface, the total is negative. To end the refund,
+process the payment and validate it.
+
+.. image:: media/getting_started_07.png
+    :align: center
+    :alt: Giving back money to customer.
+
+Close the PoS session
+---------------------
+
+At the end of the day, you will close your PoS session. To do this,
+click on the close button on the upper right of your screen and confirm.
+Now, close the session from the dashboard.
+
+.. image:: media/getting_started_08.png
+    :align: center
+    :alt: How to close a point of sale session.
+
+.. tip::
+   It’s strongly advised to close your PoS session at the end of each day.
+
+Then, you can see a summary of all transactions per payment method.
```

Comment:
```suggestion
| Once a session is closed, you can see a summary of all transactions per payment method.
| Click on a line to see all orders that were paid during your PoS session. If everything is correct, validate the session and post the closing entries.
```

simple past because is not happening anymore.